lib/JSON/Validator/Schema.pm

```diff
@@ -867,7 +867,7 @@ JSON::Validator::Schema - Base class for JSON::Validator schemas
   my $store = JSON::Validator::Store->new;
   my $schema = JSON::Validator::Schema::Draft7->new(store => $store);
 
-  # Will not fetch the fike from web, if the $store has already retrived
+  # Will not fetch the fike from web, if the $store has already retrieved
```

Comment:
Maybe fix "fike" as well 😄

Reply:
Will do. I'm sure that can be added to lintian.